oidc-ui/Dockerfile

```
@@ -0,0 +1,133 @@
+    && chown -R ${container_user}:${container_user} /usr/share/nginx /var/run/nginx /var/tmp/nginx ${i18n_path} ${theme_path} ${image_path} ${plugins_path} ${plugins_path}/temp
+
+ADD configure_start.sh configure_start.sh
+
+RUN chmod +x configure_start.sh
+
+COPY ./nginx/nginx.conf /etc/nginx/nginx.conf
+
+# copy build files to nginx html directory
+COPY --from=build_esignet_ui /app/dist  ${work_dir}
+
+RUN echo "DEFAULT_THEME=$DEFAULT_THEME" >> ${work_dir}/env.env \
```

Hadolint (2.14.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)